contracts/DelayView.sol

```diff
@@ -56,21 +56,27 @@ contract DelayView is DelayManageable {
     function GetMinDelays(address _token, uint256 _amount)
         public
         view
-        returns (uint256 _startDelay, uint256 _finishDelay)
+        returns (
+            uint256 _startDelay,
+            uint256 _cliffDelay,
+            uint256 _finishDelay
+        )
     {
         if (
             DelayLimit[_token].Amounts.length == 0 ||
             DelayLimit[_token].Amounts[0] > _amount
-        ) return (0, 0);
+        ) return (0, 0, 0);
         uint256 tempAmount = 0;
         _startDelay = DelayLimit[_token].StartDelays[0];
+        _cliffDelay = DelayLimit[_token].CliffDelays[0];
         _finishDelay = DelayLimit[_token].FinishDelays[0];
         for (uint256 i = 0; i < DelayLimit[_token].Amounts.length; i++) {
             if (
                 _amount >= DelayLimit[_token].Amounts[i] &&
                 tempAmount < DelayLimit[_token].Amounts[i]
```

Comment:
What is the purpose of `tempAmount`? This line will always be true because `DelayLimit[_token].Amounts` is ordered.